pkg/dockerregistry/server/auth_test.go

```diff
@@ -416,13 +440,13 @@ func TestAccessController(t *testing.T) {
 			expectedActions = []string{}
 		}
 		if !reflect.DeepEqual(actions, &expectedActions) {
-			t.Errorf("%s: expected\n\t%#v\ngot\n\t%#v", k, &expectedActions, actions)
+			t.Errorf("\n%s:\n expected:\n\t%#v\ngot:\n\t%#v", k, &expectedActions, actions)
```

Comment:
> @miminar but it will increase the number of packages we import from kube into registry :-) not sure if it is worth.

Every project is allowed to depend on `k8s.io/apimachinery`.